pkgs/applications/video/handbrake/default.nix

```diff
@@ -26,7 +26,7 @@
 , numactl
 , writeText
   # Processing, video codecs, containers
-, ffmpeg_5-full
+, ffmpeg-full
```

Comment:
Isn't the default standard ffmpeg not enough? And if so why?

Reply:
See https://github.com/NixOS/nixpkgs/issues/271863 for context